battery-service/src/lib.rs

```diff
@@ -41,6 +45,7 @@ pub struct Service<
     pub endpoint: comms::Endpoint,
     pub charger: charger::Charger<SmartCharger>,
     pub fuel_gauge: fuel_gauge::FuelGauge<SmartBattery>,
+    policy_signal: Signal<NoopRawMutex, MilliAmps>,
```

Comment:
Would it make sense to signal with the entire `PowerCapability` since the charger can set both current and voltage?